scripts/azureml-assets/azureml/assets/release/group_test.py

```diff
@@ -40,42 +81,75 @@
     my_env['subscription_id'] = subscription_id
     my_env['resource_group'] = resource_group
     my_env['workspace'] = workspace
+    my_env['token'] = args.token
+    if args.version_suffix:
+        my_env['version_suffix'] = args.version_suffix
     ml_client = MLClient(DefaultAzureCredential(), subscription_id, resource_group, workspace)
     submitted_job_list = []
     succeeded_jobs = []
     failed_jobs = []
+    test_coverage = defaultdict(list)
+    covered_assets = []
+
     if group_pre:
         check_call(f"python {group_pre}", env=my_env, shell=True)
 
+    pytest_jobs = defaultdict(list)  # pytest job path -> assets coverage dict
     with open(tests_dir / TEST_YML) as fp:
         data = yaml.load(fp, Loader=yaml.FullLoader)
         for job, job_data in data[test_group]['jobs'].items():
-            if 'pre' in job_data:
-                print(f"Running pre script for {job}")
-                proc = check_call(f"python3 {tests_dir / job_data['pre']}", env=my_env, shell=True)
-            print(f'Loading test job {job}')
-            test_job = azure.ai.ml.load_job(tests_dir / job_data['job'])
-            print(test_job)
-            print(f'Running test job {job}')
-            test_job = ml_client.jobs.create_or_update(test_job)
-            print(f'Submitted test job {job}')
-            print(f"Job id: {test_job.id}")
-            submitted_job_list.append(test_job)
+            if 'pytest_job' in job_data:
+                pytest_jobs[tests_dir / job_data['pytest_job']] = job_data["assets"]
```

Comment:
Consider being more consistent in use of single quotes for key names:
```suggestion
                pytest_jobs[tests_dir / job_data['pytest_job']] = job_data['assets']
```